### PR TITLE
Button bar updates

### PR DESCRIPTION
**Pull request by mikearildbrown**
Ensure it fits the new pattern framework

- [ ] Text headers ok
- [ ] Text body ok
- [ ] Existing images match guidelines
- [ ] Any extra images added

### CODE REVIEW

**Comment by mikearildbrown** [comment]
once the anatomy image is updated we should be able to merge

File: site/docs/patterns/button-bar.mdx [hunk 1]
```diff
@@ -6,44 +6,63 @@ aliases:
...
 ---
 
 A button bar displays possible actions for users to take when they reach the end of a task. You can use it to provide consistent design when implementing actionable interactions within pages or dialogs across your applications.
 
   <Diagram
-    src="/img/patterns/button-bar/with-secondary.png"
-    alt="Example button bar, with CTA to the left then primary, on the right side there are two secondary buttons"
+    src="/img/patterns/button-bar/button-bar-hero-shot.png"
+    alt="Example button bar, with CTA to the left then primary. On the right side there are two secondary buttons."
     contentPosition="top"
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:20]:
```suggestion
    border
```

File: site/docs/patterns/button-bar.mdx [hunk 2]
```diff
@@ -65,86 +84,83 @@ There are three button variants to choose from. Choosing the correct button depe
...
+In the below example:
+
     - The **Submit** primary action uses the CTA button
     - The **Cancel** alternate action uses the primary button
-    </Diagram>
+
   <Diagram
-    src="/img/patterns/button-bar/ex-2.png"
-    alt="Example showing button bar with next button on right as CTA, previous as primary to the left of next, and cancel as secondary to the left of cancel"
+    src="/img/patterns/button-bar/button-options-example-1.png"
```
Comment (joshwooding) [inline site/docs/patterns/button-bar.mdx:93]: This image and the one below are broken because the file begins with a capital

Comment (joshwooding) [reply site/docs/patterns/button-bar.mdx:93]: I've renamed the files

**Review by joshwooding — APPROVED** [review]

**Review by yunjungyeh — APPROVED** [review]

**Pull request merged**